pkg/component/crddeployer/crd.go

```diff
@@ -66,7 +66,7 @@ func (c *crdDeployer) Deploy(ctx context.Context) error {
 
 				crd.Spec = desiredCRD.Spec
 				return nil
-			})
+			}, controllerutils.SkipEmptyPatch{})
```

Comment:
There is: https://github.com/gardener/gardener/blob/62ce73bd39cc2ff5ae8d711ce5d66f80cbbe2d00/docs/development/kubernetes-clients.md#L352

But I am not aware of any webhook for CRDs. Additionally, you shared that the Deploy of the CRDs is not a cheap operation. Hence, I am fine skipping empty patches here.

cc @timebertt @rfranzke